.github/workflows/e2e_ios.yml

```diff
@@ -134,16 +132,10 @@ jobs:
         with:
           name: detox-artifacts-${{ runner.os }}-${{ github.run_id }}
           path: VAMobile/artifacts
-      
-      - name: Upload e2e-junit
-        uses: actions/upload-artifact@v3
-        with:
-          name: e2e-junit
-          path: VAMobile/e2e/test_reports/e2e-junit.xml
 
   send_test_results_to_testrail:
-    if: (github.event_name == 'workflow_run' && '!cancelled()' && github.event.workflow_run.conclusion != 'skipped') || ${{ github.event.inputs.run_testRail == true}}
-    needs: [e2e-ios]
+    if: ${{ always() }}
```

Comment:
same with this if statement

Reply:
This is necessary. If you take a look at the link I have in the description, there is currently some github actions logic that will cause it not to run this job without that statement when start_slack_job is skipped (ie when we run it manually).